securedrop/requirements/python3/translation-requirements.txt

```diff
@@ -4,9 +4,14 @@
 #
 #    pip-compile --allow-unsafe --generate-hashes --output-file=requirements/python3/translation-requirements.txt requirements/python3/translation-requirements.in
 #
-babel==2.9.1 \
-    --hash=sha256:ab49e12b91d937cd11f0b67cb259a57ab4ad2b59ac7a3b41d6c06c0ac5b0def9 \
-    --hash=sha256:bc0c176f9f6a994582230df350aa6e05ba2ebe4b3ac317eab29d9be5d2768da0
+babel==2.12.1 \
+    --hash=sha256:b4246fb7677d3b98f501a39d43396d3cafdc8eadb045f4a31be01863f655c610 \
+    --hash=sha256:cc2d99999cd01d44420ae725a21c9e3711b3aadc7976d6147f622d8581963455
+    # via
+    #   -r requirements/python3/translation-requirements.in
+    #   babelgladeextractor
+babelgladeextractor==0.7.0 \
```

Comment:
This should also get a diff review since it's being run in the build container.

Reply:
Done in <https://github.com/freedomofpress/securedrop-builder/wiki/BabelGladeExtractor-0.7.0>.